### ISSUE TITLE
Fast Track crash

### ISSUE THREAD

**Issue by DarkEviL88** [peterhaneve/ONIMods#388]
Game crashes when destroying the command module of the rocket. I tried disassembling everything I built inside the rocket first and the game still crashed out. Perhaps some incompatibility with the mod because red was highlighted first 2 mods then 3 or it is a recent hotfix. The first crash the game highlighted red only Pipe Flow Overlay and Fast Track, when I disassembled the command module without disassembling the insides, and the second crash additionally highlighted red mod Airlock Door although I did not even build it.

![image](https://github.com/peterhaneve/ONIMods/assets/50534207/21cc4745-0f5b-4c69-ab73-3d33927319b8)

Unable to teleport Линдси (Строитель) stuck on -120

at UnityEngine.Debug.LogError (System.Object message) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0
at Debug.LogError (System.Object obj) [0x00000] in :0
at FallMonitor+Instance.TeleportInWorld (System.Int32 cell) [0x00000] in :0
at FallMonitor+Instance.FallMonitor+Instance.UpdateFalling_Patch0 (FallMonitor+Instance ) [0x00000] in :0
at FallMonitor+<>c.b__12_0 (FallMonitor+Instance smi, System.Single dt) [0x00000] in :0
at BucketUpdater1[DataType].Update (DataType data, System.Single dt) [0x00000] in <d2ff0ec32ae74bb091b2c7c87705ee3b>:0  at UpdateBucketWithUpdater1[DataType].Update (System.Single dt) [0x00000] in :0
at StateMachineUpdater+BucketGroup.AdvanceOneSubTick (System.Single dt) [0x00000] in :0
at StateMachineUpdater.AdvanceOneSimSubTick () [0x00000] in :0
at Game.SimEveryTick (System.Single dt) [0x00000] in :0
at Game.Game.Update_Patch3 (Game ) [0x00000] in :0
Build: U48-568201-S

[Player.log](https://github.com/peterhaneve/ONIMods/files/12267817/Player.log)

**Comment by DarkEviL88** [comment]
I disabled Fast Track and Airlock Door disassembled the command module and the game didn't crash

**Comment by peterhaneve** [comment]
This looks like an incompatibility between multiple mods, as it does not reproduce no matter what I try. The crash appears to be when a Duplicant gets stuck out of bounds; try emptying all crew members from the rocket before deconstructing.

It could also be a base game bug, see: https://forums.kleientertainment.com/klei-bug-tracker/oni_so/game-crashed-when-deleting-spacefarer-module-with-dupe-inside-r35355/.

**Comment by DarkEviL88** [comment]
Right, I built a small command module, assigned a crew member, changed my mind and tried to break the module to build a larger one and forgot to remove the crew assignment. I heard about this bug, I just haven't encountered it before, I haven't built thin missiles before and maybe the duplicates couldn't reach the ladder and the crash occurred because they disappeared with the module, well it seemed to me that way a second before the crash. On the previous colony I did not crash because of the disassembly of rockets I built rockets and disassembled several times, but the rocket was wide and ladders on the sides were available. Fun game, but how many more bugs are there in it.... Maybe it's a coincidence, but when I turned off Fast Track and tried to disassemble the module, there was no crash, maybe there really are conflicts in the set of my mods.
